# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/angle/update-angle.py
# Commit: 77cee77c0307
# Full Hash: 77cee77c0307c253ede7cddfb385e0a08ebf3f39
# Author: Miko Mynttinen <mikokm@gmail.com>
# Date: 2019-11-11 21:52:52
# Regressor Bug: 1578576
# File Overlap Count: 1
# Description:
#   Bug 1578576 - Part 1: Update angle updater and merge base r=jgilbert
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D44561
# ==============================================================================

diff -r c1063d960295 -r 77cee77c0307 gfx/angle/update-angle.py
--- a/gfx/angle/update-angle.py	Mon Nov 11 15:39:21 2019 +0000
+++ b/gfx/angle/update-angle.py	Fri Nov 08 16:41:42 2019 +0000
@@ -144,7 +144,8 @@
 GN_ARGS = b'''
 # Build arguments go here.
 # See "gn args <out_dir> --list" for available build arguments.
-is_clang = false
+is_clang = true
+is_debug = false
 angle_enable_gl = false
 angle_enable_gl_null = false
 angle_enable_null = false
@@ -339,6 +340,7 @@
     return
 
 REGISTERED_DEFINES = {
+    'ANGLE_CAPTURE_ENABLED': True,
     'ANGLE_EGL_LIBRARY_NAME': False,
     'ANGLE_ENABLE_D3D11': True,
     'ANGLE_ENABLE_D3D9': True,
@@ -410,6 +412,12 @@
     '_WIN32_WINNT': False,
     '_WINDOWS': False,
     '__STD_C': False,
+
+    # clang specific
+    'CR_CLANG_REVISION': True,
+    'NDEBUG': False,
+    'NVALGRIND': False,
+    '_HAS_NODISCARD': False,
 }
 
 # -
